pkg/workloadmeta/types.go

```diff
@@ -84,6 +84,18 @@ type Store interface {
 	// with kind KindContainerImageMetadata and the given ID.
 	GetImage(id string) (*ContainerImageMetadata, error)
 
+	// GetProcess returns metadata about a container.  It fetches the entity
+	// with kind KindProcess and the given ID.
+	GetProcess(pid string) (*Process, error)
```

Comment:
Given that we're expecting user to pass PID as parameter, shouldn't we use an `int32` as type and let the workload meta handle the conversion? This way we decouple the information the user is passing from the internal representation in the workloadmeta.

Reply:
We typically always use `string` as the ID but I think it is fine if we use an int32 here.